README.md

```diff
@@ -1 +1,246 @@
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+
+  # Ayomid s.e.p Portfolio
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [Ayomid s.e.p Portfolio](#ayomid-sep-portfolio)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 setup and mobile first Project ](#-setup-and-mobile-first-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo](#-live-demo)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 setup and mobile first Project <a name="about-project"></a>
+<!-- #Project presentation(https://www.loom.com/share/f39c71dbc7814c3283287712da5e8f9e) -->
+
+<!-- My first major project at Microverse -->
+This is a milestone project which means features will be added in phases.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Markdown</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>No server side tech used</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>No Database used</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+<ul>
+<li>Represent my recent works</li>
+<li>Represent my skills and used technologies</li>
+<li>My contact info😎</li>
+<li>Mobile and desktop responsiveness</li>
+</ul> 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo 
+
+
+
+N/A
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+### Prerequisites
+
+In order to run this project you need:
+To clone or fork and run it in a browser
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/Paulinelali/microverse-personal-portfolio
+```
+
+### Install
+
+*N/A*
+<!-- Install this project with:
+
+
+```sh
+  cd my-project
+  gem install
+``` -->
+
+
+### Usage
+
+To run the project, do the following:
+
+Open the the folder where you've cloned the project
+Open the html file in your browser
+
+
+### Run tests
+
+To run tests, run the following command:
+
+No test required for this project
```

Comment:
- [ ] To make your documentation be more descriptive and professional, I think it would be better if you improve your `Getting started` section with correct and all information needed by users(e.g.: non-technical users,...) such as of how to `clone` your repo, `access` the cloned directory, `install` application depencies(if exist), `run/use` your app locally,... To help you, I suggest you refer this **[example file](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)**. 
**For any section that doesn't have content related to your to be filled in, make sure you remove it and you can add it later if you get information to fill in(e.g.: if your application is not deployed online from now, it is better to remove `## Live Demo` section and add it later when you will deploy it,...)**. Because, Leaving any section empty make your documentation to not be professional.